packages/shared/components/BalanceSummarySection.svelte

```diff
@@ -32,16 +37,28 @@
         }
     }
 
-    function handleAmount(isBaseToken: boolean, amount: number) {
-        return isBaseToken ? getAmount(amount) : getAmountMana(amount)
+    function handleAmount(isBaseToken: boolean = false, amount: number) {
+        if (!asset) {
+            return isBaseToken ? getAmount(amount) : getAmountMana(amount)
+        } else {
+            return getAmount(amount)
+        }
     }
 
-    function handleCurrencyAmount(isBaseToken: boolean, amount: number) {
-        return isBaseToken ? getCurrencyAmount(amount) : getAmountMana(amount)
+    function handleCurrencyAmount(isBaseToken: boolean = false, amount: number) {
+        if (!asset) {
+            return isBaseToken ? getCurrencyAmount(amount) : getAmountMana(amount)
+        } else {
+            return getCurrencyAmount(amount)
+        }
     }
 
     function getCurrencyAmount(amount: number): string {
-        return baseCoin ? formatCurrency(getMarketAmountFromAssetValue(amount, baseCoin)) : ''
+        if (!asset) {
```

Comment:
same, and you might as well add `baseCoin`

Reply:
Now that I think about this one, I don't think it was worth adding, but it's fine